tests/tests.rs

```diff
@@ -18,98 +18,47 @@ fn compare_decode_mime(expected: &str, target: &str) {
     assert_eq!(expected, String::from_utf8(decode_config(target, MIME).unwrap()).unwrap());
 }
 
-fn push_rand(buf: &mut Vec<u8>, len: usize) {
-    let mut r = rand::weak_rng();
-
-    for _ in 0..len {
-        buf.push(r.gen::<u8>());
-    }
-}
-
 // generate every possible byte string recursively and test encode/decode roundtrip
-fn roundtrip_append_recurse(byte_buf: &mut Vec<u8>, str_buf: &mut String, remaining_bytes: usize) {
```

Comment:
This got simplified along with its `_padding` variant into one function that takes a `Config`.